### ISSUE TITLE
Install wizard is broken with Docker installation

### ISSUE THREAD

**Issue by new-mikha** [go-gitea/gitea#12060]
- Gitea version (or commit ref): 1.12.1
- Git version: whatever comes with the Docker image
- Operating system: Ubuntu
- Database (use `[x]`): 
  - [ ] PostgreSQL
  - [x] MySQL
  - [ ] MSSQL
  - [ ] SQLite
- Can you reproduce the bug at https://try.gitea.io:
  - [ ] Yes (provide example URL)
  - [ ] No
  - [x] Not relevant
- Log gist:

## Description
I followed "Installation with Docker" page: https://docs.gitea.io/en-us/install-with-docker/
My docker-compose.yml is attached: [docker-compose.zip](https://github.com/go-gitea/gitea/files/4834583/docker-compose.zip)

The doco says "Visit http://server-ip:3000 and follow the installation wizard"

There was no installation wizard on this page though. Here is what I see:
![image](https://user-images.githubusercontent.com/5769929/85811205-c8d0d100-b7a0-11ea-99b7-ddfe2b7e40ea.png)

When I click Explore link, I see a page that looks like a wizard start:
![image](https://user-images.githubusercontent.com/5769929/85811250-ef8f0780-b7a0-11ea-9a3b-086f80d97dd4.png)

However when I click the Install button, it goes to the localhost:
![image](https://user-images.githubusercontent.com/5769929/85811411-704e0380-b7a1-11ea-83b6-48597a39ad38.png)

**Comment by mahabal** [comment]
I believe for the `Host` section you would want to do: `localhost:3306`.

**Comment by new-mikha** [comment]
Someone posted a suggestion which I can't see now, but it helped... 

So it works now for me, thanks! Great tool. 

The why it didn't kick-start, and I was very close to abandon it and switch to something else - turned out that I needed to specify the correct address in a couple of fields:

![image](https://user-images.githubusercontent.com/5769929/85838794-71048b00-b7dd-11ea-8350-c1c92ab6ac58.png)

I also noticed that sometimes this wizard starts as a first thing when I open URL for a freshly created container (I've made a few tests), but more often it doesn't and just shows the homepage (as I described above)

So I'd suggest the following:
1. The "Installation with Docker" page should mention that the Installation Wizard assumes "localhost:3000" for the end-user URL. I.e. if a specific IP and another port will be used - the user is expected to change the fields highlighted in the screenshot.
2. The Installation page need to be fixed - it's a bug that very often the home page is shown instead.
3. When (2) right above is fixed, then the Wizard should:
3.1 Put those settings on top of the page, into a separate section to highlight it - because it's the most important settings for the docker installation, it wouldn't work until set to the proper values.
3.2. The settings should not default to localhost:3000. It should suggest the location.host and location.port from the client side (using a bit of the client-side JavaScript), and never allow the whole service to run until the setting is specified & saved. It also should allow to change it easily from the web UI.
